ui/src/commonMain/kotlin/com/materiiapps/gloom/ui/widgets/profile/ContributionGraph.kt

```diff
@@ -25,33 +25,34 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.compositeOver
-import androidx.compose.ui.platform.LocalContext
-import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
-import com.materiiapps.gloom.R
+import com.materiiapps.gloom.Res
+import com.materiiapps.gloom.domain.manager.ToastManager
 import com.materiiapps.gloom.gql.fragment.Contributions
 import com.materiiapps.gloom.gql.type.ContributionLevel
-import com.materiiapps.gloom.utils.TimeUtils.format
-import com.materiiapps.gloom.utils.showToast
+import com.materiiapps.gloom.ui.utils.format
+import com.materiiapps.gloom.ui.utils.getPluralString
+import dev.icerock.moko.resources.compose.stringResource
+import org.koin.androidx.compose.get
 
 @Composable
 fun ContributionGraph(
     calendar: Contributions.ContributionCalendar,
     spacing: Dp = 5.dp
 ) {
+    val toastManager: ToastManager = get()
```

Comment:
:husk:
viewmodel when

Reply:
i don't like passing things down like that

Reply:
doesnt have to be passed down but oki